docs/docs/responses.mdx

```diff
@@ -164,12 +164,35 @@ Each button in the list of `buttons` should have two keys:
 * `title`: The text displayed on the buttons that the user sees.
 * `payload`: The message sent from the user to the assistant when the button is clicked.
 
+If you would like the buttons to also pass entities to the assistant:
+
+```yaml-rasa {4-8} title="domain.yml"
+responses:
+  utter_greet:
+  - text: "Hey! Would you like to purchase motor or home insurance?"
+    buttons:
+    - title: "Motor insurance"
+      payload: '/inform{{"insurance":"motor"}}'
+    - title: "Home insurance"
+      payload: '/inform{{"insurance":"home"}}'
+```
+Passing multiple entities is also possible with:
+
+```
+'/intent_name{{"entity_type_1":"entity_value_1", "entity_type_2": "entity_value_2"}}'
```

Comment:
I can't think of a good example where multiple entities need to be extracted from an intent

Reply:
Ja currently me neither.
I think it would be good to mention here that variable interpolation is the reason for the needed double quotes, otherwise it seems random

Reply:
sure, will mention the reason for double quotes